jabgui/src/main/java/org/jabref/gui/walkthrough/Timeout.java

```diff
@@ -0,0 +1,23 @@
+package org.jabref.gui.walkthrough;
+
+import javafx.animation.PauseTransition;
+import javafx.util.Duration;
+
+/// Executes a runnable after a specified duration. It uses a [PauseTransition] to
+/// ensure the runnable is executed on the JavaFX Application Thread.
+public class Timeout {
```

Comment:
I do not get the meaning of the name for this class. The code looks like just a delayed execution to me. Timeout would imply sthg is being tried for sometime until time is up, then aborted.
Maybe better call it DelayedExecutor?